backend/routers/v1/inventory.py

```diff
@@ -9,19 +9,21 @@
 
 All routes are prefixed with /api/v1/inventory
 """
-from fastapi import APIRouter
+from fastapi import APIRouter, Response
```

Comment:
Not using Response so lets remove the unnecessary import
```suggestion
from fastapi import APIRouter